src/app/feed/shared/feed.model.spec.ts

```diff
@@ -81,77 +78,70 @@ describe("Service: Feed", () => {
           return connectionStatus;
         })
       };
-      let feed: Feed = new Feed();
-      feed.setAttrs({
+      this.feed.setAttrs({
         isPublisher: true,
         connection: connection
       });
 
-      let result: boolean = feed.isEnabled("audio");
+      let result: boolean = this.feed.isEnabled("audio");
       expect(connection.getConfig).toHaveBeenCalled();
```

Comment:
Moreover you can use describes like this (just an sketch to illustrate the general idea):

```javascript
describe("when audio is enabled in the connection", function() {
  let connectionStatus: any = {
    audio: true
  };

  it("returns true", function() {
    expect(this.feed.isEnabled("audio")).toBe(true);
  });
});